src/lib/dialog/dialog-config.ts

```diff
@@ -27,6 +27,9 @@ export class MdDialogConfig {
   /** The ARIA role of the dialog element. */
   role?: DialogRole = 'dialog';
 
+  /** Custom class for the dialog, */
+  dialogClass?: string = '';
```

Comment:
Probably call this `panelClass` as well (this is already `DialogConfig`, so using the word `dialog` again on a property is redundant)